examples/advanced/random_forest/job_configs/random_forest_base/app/config/config_fed_client.json

```diff
@@ -0,0 +1,34 @@
+{
+  "format_version": 2,
+
+  "executors": [
+    {
+      "tasks": [
+        "train"
+      ],
+      "executor": {
+        "id": "Executor",
+        "path": "higgs_executor.FedXGBTreeHiggsExecutor",
+        "args": {
+          "data_split_filename": "data_split.json",
+          "training_mode": "bagging",
+          "num_client_bagging": 5,
+          "num_local_parallel_tree": 5,
+          "local_subsample": 0.8,
```

Comment:
How is this example diff. VS xgboost/tree-based algorithm?

From the config, I only see 3 differences.
On client side:
    - "num_local_parallel_tree": 5, VS 1
    - "local_subsample": 0.8,
    
On server side:
    - "num_rounds": 1, VS 101
    
    
If only differs by these 3 places, could we somehow merge these two examples? (just having 2 different base job configs)

Reply:
You are correct, after this code change they are the same now just differ in the configurations. And actually they only have 2 different base job configs, the only left "custom" code is for Higgs dataset, which we will further change to using our updated data loading scheme, will do that in a separate PR.